### ISSUE TITLE
Search results are not accessible (with keyboard navigation)

### ISSUE THREAD

**Issue by charliejhadley** [r-lib/pkgdown#2997]
As per the WCAG and POUR-CAF accessibility requirements, navigation elements should be navigable with the keyboard. However, the search results in the navigation bar do not satisfy this requirement.
